extensions/amp-analytics/analytics-vars.md

```diff
@@ -306,6 +306,24 @@ only available in a `trigger` of type `scroll`. The value of the boundary may be
 rounded based on the precision supported by the extension. For example, a
 boundary with value `1` and precision of `5` will result in value of var to be 0.
 
+##Variables as data attribute
```

Comment:
Yes, this should be up top. I would suggest:
* Keep the header and insert this right under the code example block beginning "<amp-analytics config="http://example.com/config.json">"
* Add a header before "Vars can be defined" called "Variable definitions"